main.py

```diff
@@ -22,66 +24,86 @@
 from ocpmodels.trainers import ForcesTrainer
 
 
-def main(config):
-    if args.distributed:
-        distutils.setup(config)
-
-    try:
-        setup_imports()
-        trainer = registry.get_trainer_class(config.get("trainer", "simple"))(
-            task=config["task"],
-            model=config["model"],
-            dataset=config["dataset"],
-            optimizer=config["optim"],
-            identifier=config["identifier"],
-            run_dir=config.get("run_dir", "./"),
-            is_debug=config.get("is_debug", False),
-            is_vis=config.get("is_vis", False),
-            print_every=config.get("print_every", 10),
-            seed=config.get("seed", 0),
-            logger=config.get("logger", "tensorboard"),
-            local_rank=config["local_rank"],
-            amp=config.get("amp", False),
-            cpu=config.get("cpu", False),
-        )
-        if config["checkpoint"] is not None:
-            trainer.load_pretrained(config["checkpoint"])
-
-        start_time = time.time()
-
-        if config["mode"] == "train":
-            trainer.train()
-
-        elif config["mode"] == "predict":
-            assert (
-                trainer.test_loader is not None
-            ), "Test dataset is required for making predictions"
-            assert config["checkpoint"]
-            results_file = "predictions"
-            trainer.predict(
-                trainer.test_loader,
-                results_file=results_file,
-                disable_tqdm=False,
-            )
+class SubmititRunner(submitit.helpers.Checkpointable):
```

Comment:
Since we are also using this class for local training, we should not have `Submitit` in the name. May be just `Runner`?

Reply:
Good point, done.